near-sdk-core/src/code_generator/impl_item_method_info.rs

```diff
@@ -230,9 +220,26 @@ impl ImplItemMethodInfo {
             #[cfg(not(target_arch = "wasm32"))]
             #non_bindgen_attrs
             pub fn #ident#generics(#params) #return_ident {
-                #json_args
-                #body
+                #serialize_args
+                near_sdk::PendingContractTx::new_from_bytes(&self.account_id, #ident_str, args, #is_view)
             }
         }
     }
 }
+
+fn json_serialize(attr_signature_info: &AttrSigInfo) -> TokenStream2 {
+    let args: TokenStream2 = attr_signature_info
+        .input_args()
+        .fold(None, |acc: Option<TokenStream2>, value| {
+            let ident = &value.ident;
+            let ident_str = format!("{}", ident.to_string());
```

Comment:
shouldn't `.to_string()` be enough
```suggestion
            let ident_str = ident.to_string();
```